test/helpers/cilium.go

```diff
@@ -343,14 +343,30 @@ func (c *Cilium) ServiceAdd(id int, frontend string, backends []string, rev int)
 	return c.Exec(cmd)
 }
 
+//ServiceDel delete the service ID
+func (c *Cilium) ServiceDel(id int) *CmdRes {
+	return c.Exec(fmt.Sprintf("service delete '%d'", id))
+}
+
+// ServiceList it returns the `cilium service list` output
+func (c *Cilium) ServiceList() *CmdRes {
+	return c.Exec("service list -o json")
+}
+
 //ServiceGet Get a service from cilium
 func (c *Cilium) ServiceGet(id int) *CmdRes {
 	return c.Exec(fmt.Sprintf("service get '%d'", id))
 }
 
-//ServiceDel delete the service ID
-func (c *Cilium) ServiceDel(id int) *CmdRes {
-	return c.Exec(fmt.Sprintf("service delete '%d'", id))
+// ServiceGetIds it returns and array with all services ids. If can not get the
```

Comment:
Incorrect grammar; "...returns an array with the IDs of all Cilium services. Returns an error if the IDs cannot be retrieved."